kedro-airflow/pyproject.toml

```
@@ -8,7 +8,7 @@ authors = [
     {name = "Kedro"}
 ]
 description = "Kedro-Airflow makes it easy to deploy Kedro projects to Airflow"
-requires-python = ">=3.7, <3.11"
+requires-python = ">=3.7, <=3.11"
```

Better to remove the upper cap completely, as we did with kedro and kedro-datasets

```suggestion
requires-python = ">=3.7"
```